extensions/smallrye-reactive-messaging-kafka/deployment/src/test/java/io/quarkus/smallrye/reactivemessaging/kafka/deployment/ReflectiveClassForValueSerializerPaylaodTest.java

```diff
@@ -0,0 +1,335 @@
+package io.quarkus.smallrye.reactivemessaging.kafka.deployment;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.groups.Tuple.tuple;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.apache.kafka.common.serialization.StringSerializer;
+import org.assertj.core.groups.Tuple;
+import org.eclipse.microprofile.config.Config;
+import org.eclipse.microprofile.reactive.messaging.Channel;
+import org.eclipse.microprofile.reactive.messaging.Emitter;
+import org.eclipse.microprofile.reactive.messaging.Incoming;
+import org.eclipse.microprofile.reactive.messaging.Message;
+import org.eclipse.microprofile.reactive.messaging.Outgoing;
+import org.jboss.jandex.AnnotationInstance;
+import org.jboss.jandex.IndexView;
+import org.jboss.jandex.Indexer;
+import org.jboss.jandex.Type;
+import org.junit.jupiter.api.Test;
+import org.reactivestreams.Publisher;
+
+import io.quarkus.kafka.client.serialization.JsonbDeserializer;
+import io.quarkus.kafka.client.serialization.ObjectMapperSerializer;
+import io.quarkus.smallrye.reactivemessaging.kafka.deployment.DefaultSerdeConfigTest.JacksonDto;
+import io.quarkus.smallrye.reactivemessaging.kafka.deployment.DefaultSerdeConfigTest.JacksonDtoDeserializer;
+import io.quarkus.smallrye.reactivemessaging.kafka.deployment.DefaultSerdeConfigTest.JsonbDto;
+import io.quarkus.smallrye.reactivemessaging.kafka.deployment.DefaultSerdeConfigTest.JsonbDtoSerializer;
+import io.smallrye.config.SmallRyeConfigBuilder;
+import io.smallrye.config.common.MapBackedConfigSource;
+import io.smallrye.mutiny.Multi;
+import io.smallrye.reactive.messaging.MutinyEmitter;
+import io.smallrye.reactive.messaging.kafka.KafkaRecord;
+import io.smallrye.reactive.messaging.kafka.Record;
+
+class ReflectiveClassForValueSerializerPaylaodTest {
```

Comment:
```suggestion
class ReflectiveClassForValueSerializerPayloadTest {
```

:-)

Reply:
It took me 3 minutes and numerous ctrl-f to find that out :)

Reply:
I'm blind to my own typos, but can spot others' really quickly :rofl: